jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/ArtifactUtil.java

```diff
@@ -0,0 +1,71 @@
+package org.eclipse.jkube.generator.webapp;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import org.eclipse.jkube.kit.common.PrefixedLogger;
+
+public class ArtifactUtil {
```

Comment:
Please move this class to [common util](https://github.com/eclipse/jkube/tree/master/jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util) package